CargoTransportService/AirTransport.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace CargoTransportService;
+
+public class AirTransport : CargoTransport
+{
+	private static AirTransport _instance;
```

Comment:
where you create an instance ? 
it is null here. and you are returning null object.